<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/logo.png</normaloff>img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>451</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_select_directory">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Select directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>447</width>
          <height>313</height>
         </rect>
        </property>
        <widget class="QListWidget" name="list_songs">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>451</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QToolButton" name="btn_next">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777214</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/skip-forward.svg</normaloff>icons/skip-forward.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="btn_back">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777214</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/skip-back.svg</normaloff>icons/skip-back.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_play">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/play.svg</normaloff>icons/play.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_volume">
         <property name="styleSheet">
          <string notr="true">font-size: 15px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>25</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_volume">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background: none;
border: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/volume-2.svg</normaloff>icons/volume-2.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volume_slider">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
